pkg/server/sessions/manager.go

```diff
@@ -105,7 +106,15 @@ func (sm *manager) NewSession(user *auth.User, db database.DB) (*Session, error)
 
 	sessionID := base64.URLEncoding.EncodeToString(randomBytes)
 
-	sm.sessions[sessionID] = NewSession(sessionID, user, db, sm.logger)
+	var maxActiveTxn int
+	if db != nil {
```

Comment:
are we currently supporting opening a session without a db?

Reply:
There are tests for it, which were failing